Buildings/Fluid/AirFilters/Empirical.mo

```diff
@@ -0,0 +1,315 @@
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-64,-26},{-58,-32}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-48,-30},{-42,-36}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-60,-52},{-54,-58}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{38,38},{44,32}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{40,-16},{46,-22}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-145,-100},{155,-140}},
+          textColor={0,0,255},
+          textString="%name")}),
+Diagram(coordinateSystem(preserveAspectRatio=false)),
+Documentation(info="<html>
+<p>
+An empirical model of air filters, which considers the impacts of the contaminant
+accumulation on the pressure drop and the filtration efficiency.
+</p>
+<p>
+This model does not require detailed information of filters, such as filter type
+or geometric data. Instead, its dynamic characteristics are defined by three
+parameters, <code>mCon_nominal</code>,<code>epsFun</code>, and <code>b</code>.
+</p>
+<ul>
+<li>
+The <code>mCon_nominal</code> determines the maximum mass of the contaminants that the
+filter can capture.
+</li>
+<li>
+The <code>epsFun</code> is a vector of coefficients that determines how the
+filtration efficiency changes with the contaminant accumulation.
+</li>
+<li>
+The <code>b</code> is a constant that determines how the flow coefficient changes
+with the contaminant accumulation.
+</li>
+</ul>
+<p>
+See more detailed descriptions in
+<a href=\"modelica://Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency\">
+Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency</a> and
+<a href=\"modelica://Buildings.Fluid.AirFilters.BaseClasses.FlowCoefficientCorrection\">
+Buildings.Fluid.AirFilters.BaseClasses.FlowCoefficientCorrection</a>.
+</p>
+<p>
+The input boolean flag, <code>uRep</code>, triggers the filter replacement.
+When <code>uRep</code> changes from <code>false</code> to <code>true</code>, the
+mass of the captured contaminant becomes zero.
+</p>
+<b>Note:</b>
+<p>
+A warning will be triggered when the captured contaminant mass becomes greater than the
+maximum contaminant mass (<code>mCon_nominal</code>).
+In addition, the <code>extraPropertiesNames</code> has to be defined in the medium model.
```

Comment:
Can we implement a warning? See if `Buildings.Fluid.Sensors.TraceSubstances` gives any hint.